repair/repair.cc

```diff
@@ -2573,11 +2578,19 @@ future<> repair::tablet_repair_task_impl::run() {
                         error = std::move(ep);
                     }
                 }
+                auto current = flush_times[this_shard_id()];
+                auto time = task_impl_ptr->get_flush_time();
+                flush_times[this_shard_id()] = current == gc_clock::time_point() ? time : std::min(current, time);
             }
             if (error) {
                 co_await coroutine::return_exception_ptr(std::move(error));
             }
         }).get();
+        for (auto& time : flush_times) {
+            if (time != gc_clock::time_point()) {
```

Comment:
Perhaps we should initialize the time to the time the table is created, so we don't have to do this.

Reply:
The flush time could be a bit smaller than the table create time because the cache of the batchlog and hint flush. I think it is better not bring another time in the calculation. The check is just checking if the flush time is a valid.